src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements Drawable, AreaCalculable {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, String color) {
+        super.setColor(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = firstLeg;
+    }
+
+    @Override
+    public double area() {
+        return firstLeg * secondLeg * 0.5;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rightTwriangle, area:" + area() + "sq. units, firstLeg: "
+                + firstLeg + "secondLeg: " + secondLeg
+                + " color: " + getColor());
```

Comment:
There's a typo in the string 'rightTwriangle' and missing spaces before 'secondLeg:'. It should be 'right triangle'. Also, for better readability, it's recommended to use String.format or a StringBuilder for complex string concatenations.